addons/web/static/tests/views/graph_tests.js

```diff
@@ -750,6 +750,27 @@ QUnit.module('Views', {
         graph.destroy();
     });
 
+    QUnit.test('graph view with groupBys attribute', async function (assert) {
+        assert.expect(4);
+
+        const graph = await createView({
+            View: GraphView,
+            model: "foo",
+            data: this.data,
+            arch: `
+                <graph groupBys="product_id,date">
+                    <field name="bar"/>
+                </graph>`,
+        });
+
+        assert.strictEqual(graph.renderer.props.groupBy[0], "product_id", "the groupBy field should contain correct value");
+        assert.strictEqual(graph.renderer.props.groupBy[1], "date", "the groupBy field should contain correct value");
```

Comment:
Don't say "correct value" in an assertion failure text, use instead directly what value you expected. If it fails, the sentence is clearer.